lapce-ui/src/tab.rs

```diff
@@ -1223,6 +1223,20 @@ impl LapceTab {
                     LapceUICommand::UpdateKeymap(keymap, keys) => {
                         KeyPressData::update_file(keymap, keys);
                     }
+                    LapceUICommand::OpenURI(uri) => {
+                        ctx.set_handled();
+                        if !uri.is_empty() {
+                            log::debug!(target: "lapce_ui::tab::handle_event::open_uri", "uri: {uri}");
+                            match open::that(uri) {
```

Comment:
Maybe we want to run this in a different thread

Reply:
`open::that` is non-blocking, what would be the benefit of separate thread?

Reply:
Ah OK didn't know that.